var/spack/repos/builtin/packages/sublime-text/package.py

```diff
@@ -33,8 +36,11 @@ def url_for_version(self, version):
         if version[0] == 2:
             return "https://download.sublimetext.com/Sublime%20Text%20{0}%20x64.tar.bz2".format(version)
         else:
-            return "https://download.sublimetext.com/sublime_text_{0}_x64.tar.bz2".format(version)
+            return "https://download.sublimetext.com/sublime_text_{0}_build_{1}_x64.tar.bz2".format(version[0], version[-1])
 
     def install(self, spec, prefix):
         # Sublime text comes as a pre-compiled binary.
         install_tree('.', prefix)
+
+    def setup_environment(self, spack_env, run_env):
```

Comment:
@michaelkuhn was the purpose of this to make it so that module files  generated for this package would make it available to the user?

I wanted to check since that is probably a common pattern and we might want to make to add support for doing that automatically for specially-marked packages in the future (i.e. any package marked as being a tool for end-users).

Reply:
> @michaelkuhn was the purpose of this to make it so that module files generated for this package would make it available to the user?

Exactly. There is no bin subdirectory and the main executable can be found in the prefix, so I needed to add the prefix to `PATH`.